utilities/transactions/transaction_test.cc

```diff
@@ -2823,6 +2823,93 @@ TEST_P(TransactionTest, MultiGetBatchedTest) {
   }
 }
 
+TEST_P(TransactionTest, MultiGetLargeBatchedTest) {
```

Comment:
It seems that the test is more complicated than simply using a large batch. Can you add a few inline comments of what the test does?